.github/workflows/release.yml

```diff
@@ -26,9 +29,11 @@ jobs:
       run: dotnet restore
       
     - name: Build
-      run: dotnet build --configuration Release
+      run: dotnet publish --configuration Release
     - name: Collect binaries
-      run: dotnet run --project Golem.Package --configuration Release build --dont-clean
+      run: >
+        dotnet run --project Golem.Package --configuration Release build --dont-clean 
+         --dll-file-patterns *.dll --dll-dir ${{ github.workspace }}/Golem/bin/Release/net7.0/publish
```

Comment:
I had no idea how to access `Golem` project build dir without wacky path with multiple `../`, so I created these two to control which `dll` ends up in dist package.